test/integration/build-output/test/index.test.js

```diff
@@ -178,7 +178,12 @@ describe('Build Output', () => {
           expect.stringMatching(/\/10\/10$/),
           // max of 7 preview paths
           ' [+2 more paths]',
-        ])
+        ]) {
+          // the order isn't guaranteed on the timing tests as while() is being
+          // used in the render so can block the thread of other renders sharing
+          // the same worker
+          expect(matches).toContainEqual(check)
+        }
```

Comment:
Should we sort before comparison instead of looping like this?

Reply:
pre-sorting would be similar to checking without worrying about the order so I think just checking that the expected paths are included in the output should be sufficient